content/docs/practices/histograms.md

```diff
@@ -0,0 +1,220 @@
+support only one of the two types, or they support summaries only in a
+limited fashion (lacking [quantile
+calculation](#quantiles)). [Contributions are welcome](/community/),
+of course. In general, we expect histograms to be more urgently needed
+than summaries. Histograms are also easier to implement in a client
+library, so we recommend to implement histograms first, if in
+doubt. The reason why some libraries offer summaries but not
+histograms (Ruby, the legacy Java client) is that histograms are a
+more recent feature of Prometheus.
+
+## Count and sum of observations
+
+Histograms and summaries both sample observations, typically request
+durations or response sizes. They track the number of observations
+*and* the sum of the observed values, allowing you to calculate the
+*average* of the observed values. Note that the number of observations
+(showing up in Prometheus as a time series with a `_count` suffix) is
+inherently a counter (as described above, it only goes up). The sum of
+observations (showing up as a time series with a `_sum` suffix)
+behaves like a counter, too, as long as all observations are
+positive. Obviously, request durations or response sizes are always
+positive. In principle, however, you can use summaries and histograms
+to observe negative values (e.g. temperatures in centigrade). In that
+case, the sum of observations can go down, so you cannot apply
+`rate()` to it anymore.
+
+To calculate the average request duration during the last 5 minutes
+from a histogram or summary called `http_request_duration_second`, use
+the following expression:
+
+    rate(http_request_duration_seconds_sum[5m])
+      /
+    rate(http_request_duration_seconds_count[5m])
+
+## Apdex score
+
+A straight-forward use of histograms (but not summaries) is to count
+observations falling into particular buckets of observation
+values.
+
+You might have an SLA to serve 95% of requests within 300ms. In that
+case, configure a histogram to have a bucket with an upper limit of
+0.3 seconds. You can then directly express the relative amount of
+requests served within 300ms and easily alert if the value drops below
+0.95. The following expression calculates it by job for the requests
+served in the last 5 minutes. The request durations were collected with
+a histogram called `http_request_duration_seconds`.
+
+    sum(rate(http_request_duration_seconds_bucket{le="0.3"}[5m])) by (job)
+      /
+    sum(rate(http_request_duration_seconds_count[5m])) by (job)
+
+
+You can calculate the well-known [Apdex
+score](http://en.wikipedia.org/wiki/Apdex) in a similar way. Configure
+a bucket with the target request duration as upper bound and another
+bucket with the tolerated request duration (usually 4 times the target
+request duration) as upper bound. Example: The target request duration
+is 300ms. The tolerable request duration is 1.2s. The following
+expression yields the Apdex score over the last 5 minutes:
```

Comment:
The above example is across all instances, this one is per instance. We should probably have both across all.